crypto/fipsmodule/cipher/cipher.c

```diff
@@ -225,6 +225,7 @@ int EVP_CipherInit_ex(EVP_CIPHER_CTX *ctx, const EVP_CIPHER *cipher,
 
   ctx->buf_len = 0;
   ctx->final_used = 0;
+  AES_verify_service_indicator_with_ctx(ctx);
```

Comment:
Thanks for taking care of every successful return path, Samuel. Do you know if they were all covered by the tests?